source/information-security/certificate-automation.html.md.erb

```diff
@@ -23,7 +23,17 @@ unexpected certificate expiry.
 
 ## Manual certificate requests
 
-Where automated certificate renewal is not possible, new certificates must be acquired through the DTS Operations Management team.
+If your site is being hosted on:
+
+- [Azure FrontDoor](https://docs.microsoft.com/en-us/azure/frontdoor/)
+- [Azure App Service](https://docs.microsoft.com/en-us/azure/app-service/)
+- [Azure CDN](https://docs.microsoft.com/en-us/azure/cdn/)
+
+Then you must use the [Microsoft managed certificates](https://docs.microsoft.com/en-us/azure/frontdoor/front-door-custom-domain-https#option-1-default-use-a-certificate-managed-by-front-door)
+Otherwise please raise a request in [#platops-help](https://app.slack.com/client/T1L0WSW9F/C8SR5CAMU),
+the ticket should contain the following information:
 
-To request a manually issued certificate, complete the [certificate request form](https://tools.hmcts.net/confluence/display/RD/Requesting+SSL+Certificates) and send it, 
-with a Certificate Signing Request (CSR), to Opsconfman@hmcts.net.
+- Where the certificate is to be hosted
+- DNS Name and SANS
+
+Where automated certificate renewal is not possible, new certificates must be acquired through the DTS Operations Management team.
```

Comment:
I don't think this adds anything
```suggestion
```

Reply:
how about "If your site isn't being hosted on Front Door, App Service or CDN go here - https://hmcts.github.io/information-security/certificate-automation.html#manual-certificate-requests" or something to that affect

Reply:
yeah should work, move the required section up above the heading and then make this section just about raising the request?